compiler/modules/front.syntax/parse.bal

```diff
@@ -8,8 +8,8 @@ public type FilePath record {|
 |};
 
 public function parseModulePart(string[] lines, FilePath path, int partIndex) returns ModulePart|err:Syntax {
-    SourceFile file = new(path);
-    Tokenizer tok = new (lines, file);
+    SourceFile file = new(path, scanLines(lines));
```

Comment:
This should be factored into a function

function createSourceFile(string[] lines, FilePath path) returns SourceFile